app/web_framework/cluster_slave_application.py

```diff
@@ -116,6 +126,82 @@ def get(self, build_id, subjob_id):
         self.write(response)
 
 
+class _AtomsHandler(_ClusterSlaveBaseAPIHandler):
+    def get(self, build_id, subjob_id):
+        response = {
+            'comment': 'not implemented',
+            }
+        self.write(response)
+
+
+class _AtomHandler(_ClusterSlaveBaseAPIHandler):
+    def get(self, build_id, subjob_id, atom_id):
+        response = {
+            'comment': 'not implemented',
+            }
+        self.write(response)
```

Comment:
I wasn't quite sure what to put here for these two endpoints (_AtomsHandler and _AtomHandler) that do nothing, so I just copied what we have for these types of non-implemented endpoints.